### PR TITLE
Refactor/single view for both apps

### PR DESCRIPTION
**Pull request by bob-voorhoede**
Views are now shared between the web-app and the browser-extension.
Extra:
- Shared routes file
- All head data passed to view as prop
- Fixed issue where jsonld data was not available on first load of browser ext

## Related Trello ticket(s)
- https://trello.com/c/Z7Tj80Lw/154-refactor-single-view-for-both-web-app-and-browser-extension

## Checks
- [ ] Checked browser extension (light & dark theme).
- [ ] Checked web app.

### CODE REVIEW

File: projects/browser-extension/vue-app/src/components/app-shell/app-shell.vue [hunk 1]
```diff
@@ -34,7 +34,8 @@ import RefreshButton from './refresh-button';
 export default {
   components: { AppSidebar, ExternalLink, PanelSection, RefreshButton },
   computed: {
-    ...mapState([ 'head' ]),
+    // @TODO :: Clean this up
```
Comment (vvandijck) [inline projects/browser-extension/vue-app/src/components/app-shell/app-shell.vue:37]: Are you making a ticket for all the TODO's?

File: projects/browser-extension/vue-app/src/store/index.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ export default createStore({
     robots: [],
     sitemaps: [],
     sitemapUrls: [],
-    theme: 'default',
+    theme: 'default', // @TODO :: is this needed?
```
Comment (vvandijck) [inline projects/browser-extension/vue-app/src/store/index.js]: Probably not, the `getTheme()` always returns a default value `light`.

File: projects/web-app/src/router/index.js [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+const routes = [
+  {
+    path: '/',
+    name: 'home',
+    component: Home,
+  },
+
+  ...sharedRoutes,
+
+  // Catch all fallback
```
Comment (vvandijck) [inline projects/web-app/src/router/index.js]: Should it be a good idea to add this to the browser extension routing as well?

Comment (bob-voorhoede) [reply projects/web-app/src/router/index.js]: Thought about it, but of course you can only navigate there with the buttons we give the user. Still, I'll add it for safety.

File: projects/browser-extension/vue-app/src/components/app-shell/app-shell.vue (hunk 1 above)
Comment (bob-voorhoede) [reply projects/browser-extension/vue-app/src/components/app-shell/app-shell.vue:37]: Yes ☺️

File: projects/web-app/src/router/index.js (hunk 3 above)
Comment (vvandijck) [reply projects/web-app/src/router/index.js]: 👍

**Review by vvandijck — APPROVED** [review]

**Pull request merged**